test/smoke/testApps/SystemExit/src/smokeTest/resources/jre.excludes.txt

```diff
@@ -0,0 +1,2 @@
+azul/zulu-openjdk:11
```

Comment:
why do we exclude these? for running locally probably?

Reply:
will remove this. Yes this is for local testing